docs/platform/fabric/fabricdev/core/fabricdev/channelprovider.go

```diff
@@ -43,7 +43,7 @@ type provider struct {
 	publisher               events.Publisher
 	hasher                  hash.Hasher
 	newVault                generic.VaultConstructor
-	tracerProvider          trace.TracerProvider
```

Comment:
I am wondering if the tracing and metrics packages are better located in `platform/common/services`. WDYT?